scss/v2/components/tile.scss

```diff
@@ -9,10 +9,29 @@
   padding-right: 16px;
   padding-bottom: 16px;
   margin: 16px 0px 2px 0px;
+
+  &:focus {
+    outline: 2px solid $sun-yellow;
+    outline-offset: 2px;
+    box-shadow: none; 
+  }
+
   @include breakpoint(sm) {
     font-size: 16px;
   }
 
+  &--highlighted-content {
```

Comment:
I think you're right that `highlighted-content` here isn't a modifier, it's probably more an element in this context.

So the height is specified because otherwise each of the tiles would only stretch as far as is needed to fill the text. I found that setting the height with the utility classes e.g. `height-md--48` wouldn't give me enough height to cover all of the text, too. So the height set in the code above covers the height required - matching it more closely with the design spec, at least

In this case, the `include breakpoint(md)` means that the height is being set for `md` and `lg` sizes. The breakpoint `md` = `(min-width: 768px)` - for mobile viewport sizes the height itself is fine. Hope that makes sense but let me know if you have any further comments

Reply:
Awesome ^^